compliance/aws/instances_without_fnm_agent/README.md

```diff
@@ -0,0 +1,92 @@
+## AWS EC2 Instances not running FlexNet Inventory Agent
+
+### What it does
+This policy checks all EC2 instances running in AWS to determine if the FlexNet Inventory Agent is running on the instance and reports on any that are missing the agent.
+The policy is a recommendation only policy, no action is taken during the Policy Escalation.
+
+### Functional Details
+The policy leverages the cloud API to get all current EC2 instances and the FlexNet Manager report (Custom view) API to get all AWS cloud instances with agent. It cross-checks the two lists to determine if any instances are running on the cloud that aren't known to FNM.  The policy matches the InstanceCloudID from FNMS and the instanceId from AWS. 
```

Comment:
Spell out FlexNet Manager System instead of use FNM or FNMS

Reply:
Made required changes